Readme.md

```diff
@@ -0,0 +1,48 @@
+# Okta API Access Management Sample Resource Server (ASP.NET Core)
+
+The code sample available in this repository demonstrate the use of [Okta API Access Management](http://developer.okta.com/docs/api/resources/oauth2.html) for authorizing access to a backend API using Okta's Authorization Servers.
+
+This ASP.NET Core OAuth 2.0 Resource Server will only accept calls to its TodoController if 
+1. the incoming requests contains the proper _Bearer_ OAuth token (i.e. properly validated as originating from the expected Okta organization, OpenID Connect client application and audience)
+2. the OAuth token includes a "todolist.read" scope generated by an Okta Authorization Server for the user performing the API access request to the TodoController.
+
+## Development environment instructions
+
+This code samples was written with Visual Studio 2015 Community Edition Update 3 and we strongly suggest that you use the same development environment (or any other paid-for edition of Visual Studio 2015).
+
+
+## Samples setup instructions
+
+1. In your Okta org, make sure OpenID Connect has been enabled. If not, please send an email to developers at okta dot com to get it enabled.
+2. Next, [create an OpenID Connect Native app](https://help.okta.com/en/prev/Content/Topics/Apps/Apps_App_Integration_Wizard.htm) with a Redirect URI value of ``http://127.0.0.1:[available_port]``, with the ``available_port`` value being a port available on your machine (so that your console app can listen for the browser response on that port. __Important note__: don't forget to assign at least one user to your new OpenID Connect app!
+3. Open the ``Okta OpenID Connect Windows Native Examples`` solution in Visual Studio 2015 and in the ``Okta OpenID Connect Console (Code Auth Flow - Native Browser)`` project, edit the __App.config__ file to set the following values:    
+     a. __``okta:OrganizationUrl``__: the full url of your Okta org (e.g. ``https://company.okta.com``)     
+     a. __``okta:AuthorizationServerUrl``__: the full url of your Okta org (e.g. ``https://company.okta.com``)   
```

Comment:
b.

Also, this does not format correctly either. Check out the preview before the next review and make sure that it is formatted the way you want it to look.